loopdb/sqlc/migrations/000011_static_address_loop_in.up.sql

```diff
@@ -0,0 +1,54 @@
+-- static_address_swaps...
+CREATE TABLE IF NOT EXISTS static_address_swaps (
+    -- swap_hash is the primary identifier of the swap
+    swap_hash BLOB PRIMARY KEY,
+
+    -- swap_invoice is the invoice that needs to be paid by the server to
+    -- complete the loop-in swap.
+    swap_invoice TEXT NOT NULL,
+
+    -- last_hop is an optional parameter that specifies the last hop to be
+    -- used for a loop in swap.
+    last_hop BLOB,
+
+    -- quoted_swap_fee is the swap fee in sats that the server returned in
+    -- the swap quote.
+    quoted_swap_fee BIGINT NOT NULL,
+
+    -- deposit_outpoints is a list of outpoints in the format txid:idx that are
+    -- used for this swap.
+    deposit_outpoints BLOB NOT NULL,
+
+    -- htlc_tx contains the htlc transaction without signatures.
```

Comment:
Where are signatures stored?

Reply:
The client does not have the ability to publish the htlc tx, only the server stores the combined sigs.
Andras' comment here https://github.com/lightninglabs/loop/pull/786#discussion_r1709828589 however makes me think that we don't need to store the tx at all.

Once the client partially signs the htlc tx, the server is able to publish it. The client must monitor the htlc tx spent even after a restart - but we don't need the htlc tx for that -  the htlc pk script is enough which we can reconstruct from the params that we saved.